.travis.yml

```diff
@@ -5,6 +5,15 @@ language: node_js
 cache:
   directories:
   - node_modules
+before_install:
+- |
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
The only time [we](https://github.com/ampproject/amphtml/blob/master/.travis.yml#L21-L25) ran into this is when there's a branch being tested. PRs worked perfectly.